pom.xml

```diff
@@ -88,8 +88,13 @@
 		<dependency>
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-xml</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>commons-logging</groupId>
+					<artifactId>commons-logging</artifactId>
+				</exclusion>
```

Comment:
It seems like thats the only lib that brought commons-logging dependency. I might have missed some, but thats ok I think. We are trying to make the use of logging API consistent around commons logging provided through jcl-over-slf4j not the direct use of commons-logging artifact. In app-starters, some starters were using slf4j and some were using commons-logging. Excluded commons-logging explicitly so that it won't be available to use at compile time.